cluster/src/assembly/resources/conf/iotdb-cluster.properties

```diff
@@ -130,7 +130,7 @@ multi_raft_factor=1
 # While mid consistency means the server will first try to synchronize with the leader,
 # but if failed(timeout), it will give up and just use current data it has cached before;
 # Weak consistency does not synchronize with the leader and simply use the local data
-# consistency_level=mid
+# consistency_level=strong
```

Comment:
The middle consistency may apply to some real applications but is not very suitable for tests because a timeout in middle consistency results in a wrong answer instead of sensible exceptions. Using middle consistency makes it extremely hard to write assertions. When assertions fail, one cannot know if it is caused by a synchronization timeout or other reasons.

Reply:
Currently if we run IT cases in the integration module, the `consistency_level` option is set to `strong` (See https://github.com/apache/iotdb/blob/master/integration/src/main/java/org/apache/iotdb/integration/env/ClusterNode.java#L111).
If we also want to use `strong` in any UT cases, we can just use `ClusterDescriptor.getInstance().getConfig().setConsistencyLevel(ConsistencyLevel.STRONG_CONSISTENCY);` to set it temporarily and reset it in the @After or @AfterClass. 
As the default value in `ClusterConfig.consistencyLevel` is `MID_CONSISTENCY`, changing the default config file may cause the value in the ccommented-out line inconsistent with the one in the code.

Reply:
ClusterConfig has also changed accordingly so there should be no consistency problem. I may revert this change, but I am afraid that the current consistency level may introduce inconvenience to the test team, especially when some correctness tests are added.

Reply:
I'm OK with the change of the default value in total. If it's changed, do we need to hint users in the release note or upgrading guides?

Reply:
I assume most users would prefer a strong consistency at default, so I think the current one is more reasonable. We previously set it to middle because we were running a performance test at that time, where middle consistency has more advantages. Also, consistency change does not involve any upgrading as it can be configured during runtime.